ios/RCTWebRTC/VideoCaptureController.m

```diff
@@ -4,32 +4,36 @@
 #import <React/RCTLog.h>
 
 
-@implementation VideoCaptureController {
-    RTCCameraVideoCapturer *_capturer;
-    NSString *_deviceId;
-    BOOL _running;
-    BOOL _usingFrontCamera;
-    int _width;
-    int _height;
-    int _fps;
-}
+@interface VideoCaptureController ()
+
+@property (nonatomic, strong) RTCCameraVideoCapturer *capturer;
+@property (nonatomic, strong) AVCaptureDeviceFormat *selectedFormat;
+@property (nonatomic, strong) AVCaptureDevice *device;
+@property (nonatomic, copy) NSString *deviceId;
+@property (nonatomic, assign) BOOL running;
+@property (nonatomic, assign) BOOL usingFrontCamera;
+@property (nonatomic, assign) int width;
+@property (nonatomic, assign) int height;
+@property (nonatomic, assign) int frameRate;
+
+@end
 
-@dynamic frameRate;
+@implementation VideoCaptureController
 
--(instancetype)initWithCapturer:(RTCCameraVideoCapturer *)capturer
-                 andConstraints:(NSDictionary *)constraints {
+- (instancetype)initWithCapturer:(RTCCameraVideoCapturer *)capturer
+                  andConstraints:(NSDictionary *)constraints {
     self = [super init];
     if (self) {
-        _capturer = capturer;
-        _running = NO;
+        self.capturer = capturer;
+        self.running = NO;
 
         // Default to the front camera.
-        _usingFrontCamera = YES;
+        self.usingFrontCamera = YES;
 
-        _deviceId = constraints[@"deviceId"];
-        _width = [constraints[@"width"] intValue];
-        _height = [constraints[@"height"] intValue];
-        _fps = [constraints[@"frameRate"] intValue];
+        self.deviceId = constraints[@"deviceId"];
+        self.width = [constraints[@"width"] intValue];
+        self.height = [constraints[@"height"] intValue];
+        self.frameRate = [constraints[@"frameRate"] intValue];
```

Comment:
General advise in ObjC is to not use properties in initializers (as those could be accessing uninitialized state).
But since their properties are synthesized the code should be fine.